src/test/java/org/jenkinsci/plugins/ghprb/GhprbRepositoryTest.java

```diff
@@ -109,6 +109,11 @@ public void setUp() throws Exception {
         doReturn(gt).when(trigger).getGitHub();
         
         given(gt.getRepository(anyString())).willReturn(ghRepository);
+
+        // Set rate limit before starting trigger
+        GHRateLimit initRateLimit = new GHRateLimit();
```

Comment:
This doesn't actually do anything.  The rate limit is set in GhprbTestUtil just before returning the trigger.